### PR TITLE
Updated cluster version

### PR DESCRIPTION
**Pull request by segabriel**

### CODE REVIEW

File: services/src/main/java/io/scalecube/services/Microservices.java [hunk 1]
```diff
@@ -113,7 +113,7 @@
   private final Consumer<ServiceDiscoveryConfig.Builder> discoveryOptions;
 
   private Microservices(Builder builder) {
-    this.id = IdGenerator.generateId();
+    this.id = UUID.randomUUID().toString();
```
Comment (dmytro-lazebnyi) [inline services/src/main/java/io/scalecube/services/Microservices.java:116]: What is the motivation for these changes?

Comment (segabriel) [reply services/src/main/java/io/scalecube/services/Microservices.java:116]: we removed utils-module from the current version of the cluster and right now I don't want to add a dependency only for id generation.

File: pom.xml [hunk 2]
```diff
@@ -50,11 +50,17 @@
...
+      <dependency>
+        <groupId>io.scalecube</groupId>
+        <artifactId>scalecube-cluster</artifactId>
+        <version>${scalecube-cluster.version}</version>
+      </dependency>
       <dependency>
-        <groupId>${project.groupId}</groupId>
-        <artifactId>scalecube-utils</artifactId>
+        <groupId>io.scalecube</groupId>
+        <artifactId>scalecube-transport</artifactId>
```
Comment (artem-v) [inline pom.xml]: Remove plz. Why it's here?

File: services-api/src/main/java/io/scalecube/services/transport/api/Address.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package io.scalecube.services.transport.api;
```
Comment (artem-v) [inline services-api/src/main/java/io/scalecube/services/transport/api/Address.java]: Move it to services-api  pzl. Let's not add additional dependencies on transport where it was absent.

File: services-discovery/pom.xml [hunk 4]
```diff
@@ -14,7 +14,10 @@
     <dependency>
       <groupId>io.scalecube</groupId>
       <artifactId>scalecube-cluster</artifactId>
-      <version>${scalecube-cluster.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.scalecube</groupId>
+      <artifactId>scalecube-transport</artifactId>
```
Comment (artem-v) [inline services-discovery/pom.xml]: Remove plz

File: pom.xml (hunk 2 above)
Comment (segabriel) [reply pom.xml]: it's the main pom and these are in `dependencyManagement`

File: services-discovery/src/main/java/io/scalecube/services/discovery/ScalecubeServiceDiscovery.java [hunk 5]
```diff
@@ -103,7 +105,10 @@ public ServiceEndpoint endpoint() {
   private ClusterConfig.Builder clusterConfigBuilder(ServiceDiscoveryConfig config) {
     Builder builder = ClusterConfig.builder();
     if (config.seeds() != null) {
-      builder.seedMembers(config.seeds());
+      builder.seedMembers(
+          Arrays.stream(config.seeds())
+              .map(address -> Address.create(address.host(), address.port()))
```
Comment (artem-v) [inline services-discovery/src/main/java/io/scalecube/services/discovery/ScalecubeServiceDiscovery.java]: Would u mind if I ask u to fully specify Address type here. Just to be in sync with line 48

**Review by artem-v — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (segabriel) [reply pom.xml]: however, yes, we can, the cluster depends on this transport
